ftw/solr/browser/search.py

```diff
@@ -164,8 +162,3 @@ def breadcrumbs(self, item):
             empty = {'absolute_url': '', 'Title': unicode('…', 'utf-8')}
             breadcrumbs = [breadcrumbs[0], empty] + breadcrumbs[-maxb+1:]
         return breadcrumbs
-
-
-class SearchBoxViewlet(common.SearchBoxViewlet, FacetMixin):
```

Comment:
@elioschmutz  There are several other packages, ex. plonetheme.onegov, which relay on this class for overrides.
We may simply add `from ftw.solr.viewlets.searchbox import SearchBoxViewlet` to this module for backwars compatibility? any other suggestion?